webapp/tests/forms/test_cookie_data.py

```diff
@@ -4,8 +4,8 @@
 import pytest
```

Comment:
Probably we should rename the file to test_cookie_storage.py, wdyt?

Reply:
done!